pkg/orchestrator/config/config.go

```diff
@@ -111,7 +111,8 @@ func (oc *OrchestratorConfig) Load() error {
 
 	oc.CollectorDiscoveryEnabled = config.Datadog.GetBool(key(orchestratorNS, "collector_discovery.enabled"))
 	oc.IsScrubbingEnabled = config.Datadog.GetBool(key(orchestratorNS, "container_scrubbing.enabled"))
-	oc.ExtraTags = config.Datadog.GetStringSlice(key(orchestratorNS, "extra_tags"))
+	// Tags configured by in extra_tags and DD_TAGS
+	oc.ExtraTags = append(config.Datadog.GetStringSlice(key(orchestratorNS, "extra_tags")), config.GetGlobalConfiguredTags(false)...)
```

Comment:
Would it make sense to copy the slice before appending? I think you are modifying the slice stored in the config here. Same thing in `dispatcher_main.go`.

Reply:
If I remember correctly, the config uses Viper which under the hood will parse the raw key at each call and generate a new array each time. I think it's safe to do it this way.